content_scripts/CameraVis.js

```diff
@@ -567,12 +572,27 @@ void main() {
             this.phone.add(mesh);
         }
 
-        update(mat, delayed) {
+        update(mat, delayed, rawMatricesMsg) {
             let now = performance.now();
             if (this.shaderMode === ShaderMode.HOLO) {
                 this.material.uniforms.time.value = window.performance.now();
             }
             this.lastUpdate = now;
+
+
+            if (rawMatricesMsg) {
+                this.material.uniforms.focalLength.value = new THREE.Vector2(
+                    rawMatricesMsg.focalLength[0],
```

Comment:
The same as for principal point, the focal length scaled by width/1920 , height/1080